packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts

```diff
@@ -88,14 +88,14 @@ const makeContractInput = async (input: CommandInput): Promise<ContractInput> =>
 }
 
 const validateInput = (input: CommandInput): boolean => {
-  if (!input.proposalId) throw new Error('A proposal ID is required. Provide it with --proposalId flag')
+  if (!input.proposalId) throw new Error('A Config Proposal ID is required. Provide it with --configProposal flag')
```

Comment:
This validation step now needs to know both about the arg it expects and an alias. The current framework is not prepared to do so and will become more complex in the future. Needs more thought.

No suggested action except to add in-line docs explaining the current situation and the problem.